src/components/switchComponent/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
```

Comment:
Note: Since we don't capitalize component folders naming the folder just "switch" will result in invalid js in the bundle as the name is derived from the folder name.

Reply:
I would even go with switchThemeComponent

Reply:
Moved the specific parts to the DevOverlay instead